src/nlp-mi/005_011.jl

```diff
@@ -15,9 +15,9 @@ function nlp_mi_005_011(
     @variable(model, y >= 0)
 
     @objective(model, Min, x + y)
-    @NLconstraint(model, y >= (x + 0.1) \ 1 - 0.5)
-    @NLconstraint(model, x >= y^(-2) - 0.5)
-    @NLconstraint(model, (x + y + 0.1) \ 4 >= 1)
+    @constraint(model, y >= (x + 0.1) \ 1 - 0.5)
+    @constraint(model, x >= y^(-2) - 0.5)
+    @constraint(model, (x + y + 0.1) \ 4 >= 1)
```

Comment:
We're not testing this because of `\`. How often is `\` actually used in a scalar NLP?

Reply:
This was never supported by JuMP, as far as I know 😆

Reply:
Ok maybe it was more about enumerating every basic function that is supported by Julia. I am find to remove it if you think it is not reasonable.

Reply:
It's skipped in the tests so not a big deal. We can leave as-is. I wondered if there was some particular application you had in mind.